examples/todos/containers/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { addTodo } from '../actions'
+import AddTodo from '../components/AddTodo'
+import Footer from '../components/Footer'
+import VisibleTodoList from './VisibleTodoList'
+import FilterLink from './FilterLink'
+
+const App = ({ dispatch }) => (
+ <div>
+   <AddTodo
+      onAddTodo={text => dispatch(addTodo(text))}
+    />
+   <VisibleTodoList/>
+   <Footer>
+     <FilterLink filter="SHOW_ALL">
+       All
+     </FilterLink>
+     {", "}
+     <FilterLink filter="SHOW_ACTIVE">
+       Active
+     </FilterLink>
+     {", "}
+     <FilterLink filter="SHOW_COMPLETED">
+       Completed
+     </FilterLink>
+   </Footer>
+ </div>
+)
+
+const AppContainer = connect()(App)
+
+export default AppContainer
```

Comment:
I would change the default export below to `export default connect()(App)` .. you don't gain anything from the extra const other than an extra, unused reference.